### ISSUE TITLE
AssertionError: 135.0 != 175.0 : Something went wrong. Standard Price should be 175.0!!!

### ISSUE THREAD

**Issue by ebirbe** [Vauxoo/addons-vauxoo#1139]
Into project TYP when we do a PR we get the next error when travis is testing the instance:

```
2016-12-17 22:31:19,693 6126 INFO openerp_test openerp.addons.stock_landed_costs_average.tests.test_stock_landed_costs_average: ======================================================================
2016-12-17 22:31:19,693 6126 ERROR openerp_test openerp.addons.stock_landed_costs_average.tests.test_stock_landed_costs_average: FAIL: test_02_contractor_landed_cost (openerp.addons.stock_landed_costs_average.tests.test_stock_landed_costs_average.TestsLandedCosts)
2016-12-17 22:31:19,693 6126 ERROR openerp_test openerp.addons.stock_landed_costs_average.tests.test_stock_landed_costs_average: Traceback (most recent call last):
2016-12-17 22:31:19,693 6126 ERROR openerp_test openerp.addons.stock_landed_costs_average.tests.test_stock_landed_costs_average: `   File "/home/travis/addons-vauxoo/stock_landed_costs_average/tests/test_stock_landed_costs_average.py", line 294, in test_02_contractor_landed_cost
2016-12-17 22:31:19,693 6126 ERROR openerp_test openerp.addons.stock_landed_costs_average.tests.test_stock_landed_costs_average: `     self.assert_average_product_avg(175.0)
2016-12-17 22:31:19,693 6126 ERROR openerp_test openerp.addons.stock_landed_costs_average.tests.test_stock_landed_costs_average: `   File "/home/travis/addons-vauxoo/stock_landed_costs_average/tests/test_stock_landed_costs_average.py", line 239, in assert_average_product_avg
2016-12-17 22:31:19,693 6126 ERROR openerp_test openerp.addons.stock_landed_costs_average.tests.test_stock_landed_costs_average: `     'Something went wrong. Standard Price should be %s!!!' % avg)
2016-12-17 22:31:19,693 6126 ERROR openerp_test openerp.addons.stock_landed_costs_average.tests.test_stock_landed_costs_average: ` AssertionError: 135.0 != 175.0 : Something went wrong. Standard Price should be 175.0!!!
2016-12-17 22:31:19,694 6126 INFO openerp_test openerp.addons.stock_landed_costs_average.tests.test_stock_landed_costs_average: Ran 3 tests in 24.672s
2016-12-17 22:31:19,694 6126 ERROR openerp_test openerp.addons.stock_landed_costs_average.tests.test_stock_landed_costs_average: FAILED
```

Looking for the date when PR's start to fail, and running tests in different stages (SHA) of this project, I detected the Issue was introduced near (maybe exactly with) commit e8b472d.

Please, @hbto, I need your opinion and advice here to fix it.

**Comment by ebirbe** [comment]
@hbto 

This specific changes that makes it fail are this:

```diff
-                if not line.move_id:
+                if not line.move_id or \
+                        line.move_id.location_id.usage == 'internal':
```

**Comment by hbto** [comment]
Hello @erickcion 
This issue is solved at https://github.com/Vauxoo/addons-vauxoo/commit/81ad287f76f0a90c4ef94fb7acb20d8bd77adf35

**Comment by ebirbe** [comment]
> This issue is solved at 81ad287

Thank you @hbto, confirmed +1
